src/current/v22.2/restore.md

```diff
@@ -389,8 +389,8 @@ RESTORE DATABASE bank FROM LATEST IN 's3://{bucket_name}?AWS_ACCESS_KEY_ID={key_
 {{site.data.alerts.end}}
 
 ### Restore with `AS OF SYSTEM TIME`
- 
-Running a backup with [revision history](take-backups-with-revision-history-and-restore-from-a-point-in-time.html) captures every change made within the garbage collection period leading up to and including the given timestamp, which allows you to restore to an arbitrary point-in-time within the revision history. 
```

Comment:
looks like some random formatting crept into the diff. was that intentional?

Reply:
Kind of.... I now have an automated thing in my VS Code to remove any trailing spaces. I didn't have that enabled for a long time, so where I have not touched a file in a while, it's automatically removing any trailing spaces I had left in. Thanks for checking!